package.json

```diff
@@ -202,6 +202,7 @@
     "tabbable": "1.1.0",
     "tar": "2.2.0",
     "tinygradient": "0.3.0",
+    "topojson-client": "3.0.0",
```

Comment:
The module doesn't seem to be much used and kept up to date ? (seems last change was half a year ago). Can we rely on it ?

Reply:
it seems legit to me. Written by the creator of d3 and frequently downloaded. It's a subset of `https://www.npmjs.com/package/topojson`, with only some conversion functions.